horace_core/sqw/PixelData/@pixfile_combine_info/private/set_pos_npix_start_.m

```diff
@@ -0,0 +1,16 @@
+function obj = set_pos_npix_start_(obj,val)
+%SET_POS_NPIX_START_ Accepts positive array of positions of pix distributon
```

Comment:
```suggestion
%SET_POS_NPIX_START_ Accepts positive array of positions of pix distribution
```

Reply:
Why is this called `set_pos_npix_start_`? Is `npix` not the number available to read in conjunction with each of the `pos_pixstart`s? I.e. together they define the range of pixels to read? In which case these are neither `pos`s nor `start`s if I'm understanding correctly.

Reply:
there is array of `npix` with starts with `pos_npix` and array of `pix` with starts with `pos_pix`. You read chunk of `npix` from all files and then corresponding chunk of `pix` from all files. Small enough to fit memory. And then combine them together.